src/main/resources/templates/db_scripts/schema.json

```diff
@@ -3,21 +3,13 @@
     {
       "tableName": "template",
       "withMetadata": true,
-      "pkColumnName": "_id",
-      "generateId": false,
       "withAuditing": false,
```

Comment:
I think you can remove flag "withAuditing" at all. It is "false" by default as described here https://github.com/folio-org/raml-module-builder#the-post-tenant-api

Reply:
I second.